### ISSUE TITLE
Thing not found

### ISSUE THREAD

**Issue by dhylands** [mozilla-iot/gateway#2087]
I added a new zwave sensor (the Aeotec MultiSensor 6). The main page shows the temperature. Clicking on the splat gives a "Thing not found error".

The Webconsole shows the following errors:
```
Thing id zwave-d7e613b4-4 not found TypeError: value.toFixed is not a function things.js:276:16
    refreshThing things.js:276
    notify es.promise.js:105
    flush microtask.js:26
```

The thing description looks like:
```
{
  "title": "zwave-d7e613b4-4",
  "type": "thing",
  "@context": "https://iot.mozilla.org/schemas",
  "@type": [
    "TemperatureSensor"
  ],
  "description": "",
  "href": "/things/zwave-d7e613b4-4",
  "properties": {
    "temperature": {
      "title": "Temperature",
      "type": "number",
      "@type": "TemperatureProperty",
      "unit": "degree celsius",
      "links": [
        {
          "rel": "property",
          "href": "/things/zwave-d7e613b4-4/properties/temperature"
        }
      ]
    },
    "luminance": {
      "title": "Luminance",
      "type": "number",
      "unit": "lux",
      "links": [
        {
          "rel": "property",
          "href": "/things/zwave-d7e613b4-4/properties/luminance"
        }
      ]
    },
    "humidity": {
      "title": "Humidity",
      "type": "number",
      "@type": "LevelProperty",
      "unit": "percent",
      "minimum": 0,
      "maximum": 100,
      "links": [
        {
          "rel": "property",
          "href": "/things/zwave-d7e613b4-4/properties/humidity"
        }
      ]
    },
    "uvIndex": {
      "title": "UV Index",
      "type": "number",
      "links": [
        {
          "rel": "property",
          "href": "/things/zwave-d7e613b4-4/properties/uvIndex"
        }
      ]
    },
    "batteryLevel": {
      "title": "Battery",
      "type": "number",
      "@type": "LevelProperty",
      "unit": "percent",
      "minimum": 0,
      "maximum": 100,
      "readOnly": true,
      "links": [
        {
          "rel": "property",
          "href": "/things/zwave-d7e613b4-4/properties/batteryLevel"
        }
      ]
    }
  },
  "actions": {},
  "events": {},
  "links": [
    {
      "rel": "properties",
      "href": "/things/zwave-d7e613b4-4/properties"
    },
    {
      "rel": "actions",
      "href": "/things/zwave-d7e613b4-4/actions"
    },
    {
      "rel": "events",
      "href": "/things/zwave-d7e613b4-4/events"
    },
    {
      "rel": "alternate",
      "mediaType": "text/html",
      "href": "/things/zwave-d7e613b4-4"
    },
    {
      "rel": "alternate",
      "href": "ws://localhost:8080/things/zwave-d7e613b4-4"
    }
  ],
  "layoutIndex": 8,
  "selectedCapability": "TemperatureSensor",
  "iconHref": null,
  "id": "http://localhost:8080/things/zwave-d7e613b4-4",
  "base": "http://localhost:8080/",
  "securityDefinitions": {
    "oauth2_sc": {
      "scheme": "oauth2",
      "flow": "code",
      "authorization": "http://localhost:8080/oauth/authorize",
      "token": "http://localhost:8080/oauth/token",
      "scopes": [
        "/things/zwave-d7e613b4-4:readwrite",
        "/things/zwave-d7e613b4-4",
        "/things:readwrite",
        "/things"
      ]
    }
  },
  "security": "oauth2_sc"
}
```

This might be because the temperature property is missing min/max, but it seems like a bad way to react to missing attributes.

**Comment by dhylands** [comment]
If I press refresh, I can see it display the thing with the bubbles for each of the properties, but it then gets replaced immediately with the "Thing not found" error page instead.

**Comment by mrstegeman** [comment]
My guess is that one of the `LevelProperty`s is getting set to null or undefined.

**Comment by dhylands** [comment]
The error is originating from here:
https://github.com/mozilla-iot/gateway/blob/f57bfd6a5e7d9dc0211f6a0917a6c9ca33ca7880/static/js/components/property/level.js#L225

value seems to be a string and toFixed doesn't exist on strings.

**Comment by mrstegeman** [comment]
Makes sense. Fixed in #2088.
